enum/Scheduler.ts

```diff
@@ -1,4 +1,5 @@
 export enum JobName {
     SESSION_MAINTENANCE = 'session-maintenance',
     EVENT_SCHEDULER =  'event-scheduler',
+    QUEUE_WINDOW_SCHEDULER =  'queue-window-scheduler',
```

Comment:
Can we rename it to something relatable? How about `CONTINUE_BLACKOUT_TIMEOUT_SCHEDULER`?